2021-01-30 19:55:33,507 INFO c.e.d.c.UserController [main] Username is set with savio
2021-01-30 19:55:33,516 INFO c.e.d.c.UserController [main] User Creation is success: User savio created with requested username, cart, and encoded password
2021-01-30 19:56:02,963 INFO c.e.d.c.UserController [main] Username is set with savio
2021-01-30 19:56:02,973 INFO c.e.d.c.UserController [main] User Creation is success: User savio created with requested username, cart, and encoded password
2021-01-30 19:56:28,963 INFO c.e.d.SareetaApplication [main] Starting SareetaApplication on P78039 with PID 36072 (/Users/ts-dom.dukuzumuremyi/Documents/Develop/Person/Udacity/JWND/e-commerce/target/classes started by ts-dom.dukuzumuremyi in /Users/ts-dom.dukuzumuremyi/Documents/Develop/Person/Udacity/JWND/e-commerce)
2021-01-30 19:56:28,968 DEBUG c.e.d.SareetaApplication [main] Running with Spring Boot v2.1.5.RELEASE, Spring v5.1.7.RELEASE
2021-01-30 19:56:28,969 INFO c.e.d.SareetaApplication [main] No active profile set, falling back to default profiles: default
2021-01-30 19:56:32,630 DEBUG c.e.d.s.JWTAuthenticationVerficationFilter [main] Filter 'JWTAuthenticationVerficationFilter' configured for use
2021-01-30 19:56:33,241 INFO c.e.d.SareetaApplication [main] Started SareetaApplication in 4.649 seconds (JVM running for 6.115)
2021-01-30 19:56:52,296 INFO c.e.d.c.UserController [main] Username is used to find a user is 
2021-01-30 19:56:52,345 INFO c.e.d.c.UserController [main] Username is set with savio
2021-01-30 19:56:52,346 INFO c.e.d.c.UserController [main] User Creation is success: User savio created with requested username, cart, and encoded password
2021-01-30 19:57:04,439 INFO c.e.d.c.ItemControllerTest [main] Starting ItemControllerTest on P78039 with PID 36097 (started by ts-dom.dukuzumuremyi in /Users/ts-dom.dukuzumuremyi/Documents/Develop/Person/Udacity/JWND/e-commerce)
2021-01-30 19:57:04,444 DEBUG c.e.d.c.ItemControllerTest [main] Running with Spring Boot v2.1.5.RELEASE, Spring v5.1.7.RELEASE
2021-01-30 19:57:04,444 INFO c.e.d.c.ItemControllerTest [main] No active profile set, falling back to default profiles: default
2021-01-30 19:57:08,827 INFO c.e.d.c.ItemControllerTest [main] Started ItemControllerTest in 4.619 seconds (JVM running for 6.936)
2021-01-30 19:57:22,961 INFO c.e.d.c.OrderControllerTest [main] Starting OrderControllerTest on P78039 with PID 36122 (started by ts-dom.dukuzumuremyi in /Users/ts-dom.dukuzumuremyi/Documents/Develop/Person/Udacity/JWND/e-commerce)
2021-01-30 19:57:22,969 DEBUG c.e.d.c.OrderControllerTest [main] Running with Spring Boot v2.1.5.RELEASE, Spring v5.1.7.RELEASE
2021-01-30 19:57:22,970 INFO c.e.d.c.OrderControllerTest [main] No active profile set, falling back to default profiles: default
2021-01-30 19:57:27,058 INFO c.e.d.c.OrderControllerTest [main] Started OrderControllerTest in 4.3 seconds (JVM running for 6.655)
2021-01-30 19:57:27,100 INFO c.e.d.c.OrderController [main] Order Request is Success and User savio was found and finally order submitted
2021-01-30 19:57:38,195 INFO c.e.d.c.UserController [main] Username is used to find a user is 
2021-01-30 19:57:38,240 INFO c.e.d.c.UserController [main] Username is set with savio
2021-01-30 19:57:38,241 INFO c.e.d.c.UserController [main] User Creation is success: User savio created with requested username, cart, and encoded password
2021-01-30 19:58:28,113 INFO c.e.d.c.UserController [main] Username is set with savio
2021-01-30 19:58:28,118 ERROR c.e.d.c.UserController [main] User Creation is failed: Password length has not met, Password length required at least 8 characters. So, Cannot create an user savio. 
2021-01-30 20:01:56,932 INFO c.e.d.c.UserController [main] Username is set with savio
2021-01-30 20:01:56,942 INFO c.e.d.c.UserController [main] User Creation is success: User savio created with requested username, cart, and encoded password
2021-01-30 20:04:45,181 INFO c.e.d.c.OrderControllerTest [main] Starting OrderControllerTest on P78039 with PID 36479 (started by ts-dom.dukuzumuremyi in /Users/ts-dom.dukuzumuremyi/Documents/Develop/Person/Udacity/JWND/e-commerce)
2021-01-30 20:04:45,189 DEBUG c.e.d.c.OrderControllerTest [main] Running with Spring Boot v2.1.5.RELEASE, Spring v5.1.7.RELEASE
2021-01-30 20:04:45,189 INFO c.e.d.c.OrderControllerTest [main] No active profile set, falling back to default profiles: default
2021-01-30 20:04:49,195 INFO c.e.d.c.OrderControllerTest [main] Started OrderControllerTest in 4.239 seconds (JVM running for 6.789)
2021-01-30 20:04:49,238 ERROR c.e.d.c.OrderController [main] Order Request Fails, because User savio was not found
2021-01-30 20:05:16,596 INFO c.e.d.c.OrderControllerTest [main] Starting OrderControllerTest on P78039 with PID 36504 (started by ts-dom.dukuzumuremyi in /Users/ts-dom.dukuzumuremyi/Documents/Develop/Person/Udacity/JWND/e-commerce)
2021-01-30 20:05:16,604 DEBUG c.e.d.c.OrderControllerTest [main] Running with Spring Boot v2.1.5.RELEASE, Spring v5.1.7.RELEASE
2021-01-30 20:05:16,605 INFO c.e.d.c.OrderControllerTest [main] No active profile set, falling back to default profiles: default
2021-01-30 20:05:20,525 INFO c.e.d.c.OrderControllerTest [main] Started OrderControllerTest in 4.164 seconds (JVM running for 6.689)
2021-01-30 20:05:20,562 INFO c.e.d.c.OrderController [main] Order Request is Success and User savio was found and finally order submitted
2021-01-30 20:18:06,367 INFO c.e.d.c.OrderControllerTest [main] Starting OrderControllerTest on P78039 with PID 37221 (started by ts-dom.dukuzumuremyi in /Users/ts-dom.dukuzumuremyi/Documents/Develop/Person/Udacity/JWND/e-commerce)
2021-01-30 20:18:06,373 DEBUG c.e.d.c.OrderControllerTest [main] Running with Spring Boot v2.1.5.RELEASE, Spring v5.1.7.RELEASE
2021-01-30 20:18:06,374 INFO c.e.d.c.OrderControllerTest [main] No active profile set, falling back to default profiles: default
2021-01-30 20:18:10,339 INFO c.e.d.c.OrderControllerTest [main] Started OrderControllerTest in 4.202 seconds (JVM running for 7.092)
2021-01-30 20:18:10,383 INFO c.e.d.c.OrderController [main] Order Request is Success and User savio was found and finally order submitted
2021-01-30 20:18:32,189 INFO c.e.d.c.OrderControllerTest [main] Starting OrderControllerTest on P78039 with PID 37246 (started by ts-dom.dukuzumuremyi in /Users/ts-dom.dukuzumuremyi/Documents/Develop/Person/Udacity/JWND/e-commerce)
2021-01-30 20:18:32,197 DEBUG c.e.d.c.OrderControllerTest [main] Running with Spring Boot v2.1.5.RELEASE, Spring v5.1.7.RELEASE
2021-01-30 20:18:32,198 INFO c.e.d.c.OrderControllerTest [main] No active profile set, falling back to default profiles: default
2021-01-30 20:18:36,213 INFO c.e.d.c.OrderControllerTest [main] Started OrderControllerTest in 4.25 seconds (JVM running for 6.768)
2021-01-30 20:18:36,258 INFO c.e.d.c.OrderController [main] Order Request is Success and User savio was found and finally order submitted
2021-01-30 20:18:44,402 INFO c.e.d.SareetaApplication [main] Starting SareetaApplication on P78039 with PID 37254 (/Users/ts-dom.dukuzumuremyi/Documents/Develop/Person/Udacity/JWND/e-commerce/target/classes started by ts-dom.dukuzumuremyi in /Users/ts-dom.dukuzumuremyi/Documents/Develop/Person/Udacity/JWND/e-commerce)
2021-01-30 20:18:44,411 DEBUG c.e.d.SareetaApplication [main] Running with Spring Boot v2.1.5.RELEASE, Spring v5.1.7.RELEASE
2021-01-30 20:18:44,413 INFO c.e.d.SareetaApplication [main] No active profile set, falling back to default profiles: default
2021-01-30 20:18:48,847 DEBUG c.e.d.s.JWTAuthenticationVerficationFilter [main] Filter 'JWTAuthenticationVerficationFilter' configured for use
2021-01-30 20:18:49,657 ERROR o.a.c.u.LifecycleBase [main] Failed to start component [Connector[HTTP/1.1-8080]]
org.apache.catalina.LifecycleException: Protocol handler start failed
	at org.apache.catalina.connector.Connector.startInternal(Connector.java:1008) ~[tomcat-embed-core-9.0.19.jar:9.0.19]
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183) [tomcat-embed-core-9.0.19.jar:9.0.19]
	at org.apache.catalina.core.StandardService.addConnector(StandardService.java:226) [tomcat-embed-core-9.0.19.jar:9.0.19]
	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.addPreviouslyRemovedConnectors(TomcatWebServer.java:259) [spring-boot-2.1.5.RELEASE.jar:2.1.5.RELEASE]
	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.start(TomcatWebServer.java:197) [spring-boot-2.1.5.RELEASE.jar:2.1.5.RELEASE]
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.startWebServer(ServletWebServerApplicationContext.java:311) [spring-boot-2.1.5.RELEASE.jar:2.1.5.RELEASE]
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.finishRefresh(ServletWebServerApplicationContext.java:164) [spring-boot-2.1.5.RELEASE.jar:2.1.5.RELEASE]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:552) [spring-context-5.1.7.RELEASE.jar:5.1.7.RELEASE]
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:142) [spring-boot-2.1.5.RELEASE.jar:2.1.5.RELEASE]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:775) [spring-boot-2.1.5.RELEASE.jar:2.1.5.RELEASE]
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:397) [spring-boot-2.1.5.RELEASE.jar:2.1.5.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:316) [spring-boot-2.1.5.RELEASE.jar:2.1.5.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1260) [spring-boot-2.1.5.RELEASE.jar:2.1.5.RELEASE]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1248) [spring-boot-2.1.5.RELEASE.jar:2.1.5.RELEASE]
	at com.example.demo.SareetaApplication.main(SareetaApplication.java:20) [classes/:?]
Caused by: java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:455) ~[?:?]
	at sun.nio.ch.Net.bind(Net.java:447) ~[?:?]
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:219) ~[?:?]
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:80) ~[?:?]
	at org.apache.tomcat.util.net.NioEndpoint.initServerSocket(NioEndpoint.java:239) ~[tomcat-embed-core-9.0.19.jar:9.0.19]
	at org.apache.tomcat.util.net.NioEndpoint.bind(NioEndpoint.java:213) ~[tomcat-embed-core-9.0.19.jar:9.0.19]
	at org.apache.tomcat.util.net.AbstractEndpoint.bindWithCleanup(AbstractEndpoint.java:1116) ~[tomcat-embed-core-9.0.19.jar:9.0.19]
	at org.apache.tomcat.util.net.AbstractEndpoint.start(AbstractEndpoint.java:1202) ~[tomcat-embed-core-9.0.19.jar:9.0.19]
	at org.apache.coyote.AbstractProtocol.start(AbstractProtocol.java:568) ~[tomcat-embed-core-9.0.19.jar:9.0.19]
	at org.apache.catalina.connector.Connector.startInternal(Connector.java:1005) ~[tomcat-embed-core-9.0.19.jar:9.0.19]
	... 14 more
2021-01-30 20:18:49,676 WARN o.a.c.l.WebappClassLoaderBase [main] The web application [ROOT] appears to have started a thread named [HikariPool-1 housekeeper] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.base@12.0.2/jdk.internal.misc.Unsafe.park(Native Method)
 java.base@12.0.2/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:235)
 java.base@12.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:2123)
 java.base@12.0.2/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:1182)
 java.base@12.0.2/java.util.concurrent.ScheduledThreadPoolExecutor$DelayedWorkQueue.take(ScheduledThreadPoolExecutor.java:899)
 java.base@12.0.2/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1054)
 java.base@12.0.2/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1114)
 java.base@12.0.2/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
 java.base@12.0.2/java.lang.Thread.run(Thread.java:835)
2021-01-30 20:18:49,677 WARN o.a.c.l.WebappClassLoaderBase [main] The web application [ROOT] appears to have started a thread named [HikariPool-1 connection adder] but has failed to stop it. This is very likely to create a memory leak. Stack trace of thread:
 java.base@12.0.2/jdk.internal.misc.Unsafe.park(Native Method)
 java.base@12.0.2/java.util.concurrent.locks.LockSupport.parkNanos(LockSupport.java:235)
 java.base@12.0.2/java.util.concurrent.locks.AbstractQueuedSynchronizer$ConditionObject.awaitNanos(AbstractQueuedSynchronizer.java:2123)
 java.base@12.0.2/java.util.concurrent.LinkedBlockingQueue.poll(LinkedBlockingQueue.java:458)
 java.base@12.0.2/java.util.concurrent.ThreadPoolExecutor.getTask(ThreadPoolExecutor.java:1053)
 java.base@12.0.2/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1114)
 java.base@12.0.2/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
 java.base@12.0.2/java.lang.Thread.run(Thread.java:835)
